ts/features/wallet/bancomatpay/component/BPayWalletPreview.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from "react";
+import { Image, ImageStyle, StyleProp } from "react-native";
+import { Option } from "fp-ts/lib/Option";
+import { connect } from "react-redux";
+import { Dispatch } from "redux";
+import bPayImage from "../../../../../img/wallet/cards-icons/bPay.png";
+import { Body } from "../../../../components/core/typography/Body";
+import { IOStyles } from "../../../../components/core/variables/IOStyles";
+import I18n from "../../../../i18n";
+import { GlobalState } from "../../../../store/reducers/types";
+import { CardPreview } from "../../component/CardPreview";
+import { navigateToBPayDetailScreen } from "../../../../store/actions/navigation";
+import { BPayPaymentMethod } from "../../../../types/pagopa";
+import { useImageResize } from "../../onboarding/bancomat/screens/hooks/useImageResize";
+
+type OwnProps = {
+  bPay: BPayPaymentMethod;
+};
+
+type Props = ReturnType<typeof mapDispatchToProps> &
+  ReturnType<typeof mapStateToProps> &
+  OwnProps;
+
+const BASE_IMG_W = 160;
+const BASE_IMG_H = 20;
+/**
+ * Render the image (if available) or the bank name (if available)
+ * or the generic bancomatPay string (final fallback).
+ * @param props
+ * @param size
+ */
+const renderLeft = (props: Props, size: Option<[number, number]>) =>
+  size.fold(
+    <Body style={IOStyles.flex} numberOfLines={1} testID={"bankLogoFallback"}>
+      {props.bPay.abiInfo?.name ??
+        props.bPay.info.bankName ??
+        I18n.t("wallet.methods.bancomatPay.name")}
```

Comment:
```suggestion
      {props.bPay.caption}
```
Added in #2685 the common logic to uniform the behaviour.